Makefile

```
@@ -65,7 +65,7 @@ container-image:
 		--build-arg HUGO_VERSION=$(HUGO_VERSION)
 
 container-build: module-check
-	$(CONTAINER_RUN) --read-only $(CONTAINER_IMAGE) hugo --minify
+	$(CONTAINER_RUN) --read-only $(CONTAINER_IMAGE) sh -c "npm ci && hugo --minify"
```

```suggestion
	$(CONTAINER_RUN) --read-only --mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 $(CONTAINER_IMAGE) sh -c "npm install -D autoprefixer postcss postcss-cli && hugo --minify"
```

but I also think that `npm install -D autoprefixer postcss postcss-cli` belongs in the `Dockerfile`.